### PR TITLE
Add desktop capture extension for Peer2peer

### PR DESCRIPTION
**Pull request by KaptenJansson**
Due to the desktop capture extension needing white listing domains I thought it would be better just to pull in the extension to this repo  so that we have full control over it.

### CODE REVIEW

**Comment by KaptenJansson** [comment]
andresusanopinto@ Do you see any issues here or can I merge this?

**Comment by andresusanopinto** [comment]
Can you state where this extension was before?

File: src/manual/peer2peer/extension/content-script.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+'use strict';
```
Comment (andresusanopinto) [inline src/manual/peer2peer/extension/content-script.js]: Missing copyright (I guess this was moved from webrtc/samples or another repository that uses exactly the same copyright we do).

File: src/manual/peer2peer/extension/background.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+'use strict';
```
Comment (andresusanopinto) [inline src/manual/peer2peer/extension/background.js]: Copyright missing.

**Comment by KaptenJansson** [comment]
It still lives in the samples repo, https://github.com/webrtc/samples/tree/master/src/content/getusermedia/desktopcapture.

I do not intend to remove it from the samples repo, rather I want to keep it there.

I want a separate copy here which we can modify as we see fit, currently only the URL white-list is the only difference.

**Comment by andresusanopinto** [comment]
lgtm

**Pull request merged**